### PR TITLE
Updated the description of DASH.md

### PR DESCRIPTION
**Pull request by Expertium**
I suggest adding DASH[ACT-R] to the table, but sorting by log loss. Pay attention to highliting the best result in bold, as the best log-loss and the best RMSE will not be in the same row anymore.

### CODE REVIEW

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Comment by Expertium** [comment]
No no, don't merge it yet. You haven't added DASH[ACT-R]. Merge it after you update all tables.

**Comment by Expertium** [comment]
Ah, I didn't click "Allow edits by maintainers". My bad.
Actually, wait, who exactly counts as a "maintainer"? Do I need to click that to allow you to edit my pull request?

**Pull request merged**
